qutebrowser/browser/qutescheme.py

```diff
@@ -425,6 +425,15 @@ def qute_settings(url):
     return 'text/html', html
 
 
+@add_handler('back')
+def qute_back(url):
+    """Handler for qute://back. Simple page to free ram / lazy load a site,
```

Comment:
Please format this according to [pep 257](https://www.python.org/dev/peps/pep-0257/):

```python
    """Handler for qute://back.
    
    Simple page to free ram / lazy load a site, goes back on focusing the tab.
    """
```